### ISSUE TITLE
The app crashes when you double-click the image copy button

### ISSUE THREAD

**Issue by Fadds** [telegramdesktop/tdesktop#24413]
### Steps to reproduce

1. Open any picture in tg-viewer
2. Press RMB on it
3. Double click "Copy" in context menu

### Expected behaviour

The picture is copied to the clipboard
OR
The image is copied to the clipboard and closed

### Actual behaviour

App crashes.
Report Tag: 032826cb-f632-4ad8-9f87-0ac4805eacfb

### Operating system

Windows 10 Pro 21H2 19044.1526

### Version of Telegram Desktop

3.7.1 x64

### Installation source

Static binary from official website

### Logs

```shell
[2022.04.22 23:02:21] Assertion Failed! "!_previousLoopNestingLevels.empty()" sandbox.cpp:536
```

**Comment by Aokromes** [comment]
cannot reproduce on windows 11...

**Comment by ashkanhasebi** [comment]
I have this problem too. on windows 11.
also, there are random frequent crashes of the app in version 3.7.1 of the desktop (64-bit Windows 11) when using the app.
reported on: [https://bugs.telegram.org/c/16710](https://bugs.telegram.org/c/16710)

**Comment by vimpostor** [comment]
This is the same problem as in #23966
The fix is very easy and slightly hacky at the same time: Patch the context menu buttons to ignore a double click.
The hacky part is that there is by default no way in Qt to ignore double click and still accept single click. However, this can easily be hacked together by starting a `QTimer` on the first click that disables the context menu for a short duration, effectively disabling the second click.

**Comment by john-preston** [comment]
This should be fixed in 3.7.2.
